Lots of small fixes (#231)
vitessio/website: content/en/docs/get-started/kubernetes.md

* You moved customer and corder from the commerce’s VSchema to customer’s VSchema. Note that the physical tables are still in commerce.
* You requested that the schema for customer and corder be copied to customer using the `copySchema` directive.

> The move in the VSchema should not be material yet because any queries sent to customer are still redirected to commerce, where all the data is still present.

Reviewer: ```suggestion
The move in the VSchema should make no difference yet because any queries sent to customer are still redirected to commerce, where all the data is still present.
```
